src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -2,6 +2,8 @@
 
 {% block content %}
 
+  {% set psStockManagement = 'PS_STOCK_MANAGEMENT'|configuration %}
```

Comment:
i'd like to see a boolean value instead of guess if 0 or 1 is expected here, don't you ?